tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -588,18 +601,20 @@
        * this does nothing.
        */
       _ensureSelectedDashboardStamped(containersStamped, activeDashboards,
-                                      dashboard) {
-        if (!containersStamped || !activeDashboards || !dashboard) {
+                                      selectedDashboard) {
+        if (!containersStamped || !activeDashboards || !selectedDashboard) {
           return;
         }
         const container = this.$$(
-          `.dashboard-container[data-dashboard=${dashboard}]`);
+            `.dashboard-container[data-dashboard=${selectedDashboard}]`);
         if (!container) {
           // This dashboard doesn't exist. Nothing to do here.
+          console.log(`no exist - ${selectedDashboard}`);
```

Comment:
Did you mean to remove this? If you want to keep it, (a) what value does it add? and (b) could you improve the message?

Reply:
Removed.